src/main/resources/app.conf

```diff
@@ -80,35 +80,46 @@ app {
       # the initial delay before retrying that will be multiplied with the 'factor' for each attempt
       initial-delay = 100 milliseconds
       initial-delay = ${?KEY_VALUE_STORE_RETRY_INITIAL_DELAY}
+      # the maximum delay applied to the retrying strategy
+      max-delay = 10 minutes
       # maximum number of retries in case of failure (applicable only for strategy "exponential")
       max-retries = 7
       max-retries = ${?KEY_VALUE_STORE_RETRY_MAX_RETRY}
-      # the exponential factor (applicable only for strategy "exponential")
-      factor = 2
+      # the exponential random factor (applicable only for strategy "exponential")
+      factor = 0.2
       factor = ${?KEY_VALUE_STORE_RETRY_FACTOR}
+      # the linear increment (applicable only for strategy "linear")
+      increment = 500 milliseconds
     }
   }
 
   indexing {
 
-    # Maximum number of events taken on each batch
-    batch = 10
-    batch = ${?INDEXING_BATCH}
-
-    # Maximum amount of time to wait for the number of events to be taken on each batch
-    batch-timeout = 400 millis
+    # time to wait before the next batch is consumed
+    batch-timeout = 40 millis
     batch-timeout = ${?INDEXING_BATCH_TIMEOUT}
 
-    retry {
-      # The maximum number of times an index function is retried
-      max-count = 10
-      max-count = ${?INDEXING_RETRY_MAX}
+    # number of events to wait before the next batch is consumed
+    batch-chunk = 10
```

Comment:
batch and chunk mean similar things; either keep the name `batch` or rename to `batch-size`